devsetup/scripts/tripleo.sh

```diff
@@ -39,43 +41,94 @@ if [[ ! -f $SSH_KEY_FILE ]]; then
     exit 1
 fi
 
+sudo dnf install -y python-jinja2
 source ${SCRIPTPATH}/common.sh
 
+# Clock synchronization is important for both Ceph and OpenStack services, so both ceph deploy and tripleo deploy commands will make use of chrony to ensure the clock is properly in sync.
+# We'll use the NTP_SERVER environmental variable to define the NTP server to use.
+# If we are running alls these commands in a system inside the Red Hat network we should use the clock.corp.redhat.com server:
+# export NTP_SERVER=clock.corp.redhat.com
+# And when running it from our own systems outside of the Red Hat network we can use any available server:
+# export NTP_SERVER=pool.ntp.org
+
+if [ $EDPM_COMPUTE_CELLS -eq 2 ] || [ $EDPM_COMPUTE_CELLS -gt 3 ] || [ $EDPM_COMPUTE_CELLS -eq 0 ] ; then
+    echo "Only a main cell0 plus a 2 additional compute cells supported yet!"
+    exit 1
+fi
+
+# TODO(bogdando): multi-cell with ceph HCI
+if [ $EDPM_COMPUTE_CELLS -eq 3 ] && [[ "$EDPM_COMPUTE_CEPH_ENABLED" =~ "rue" ]]; then
```

Comment:
wouldn't it be better to stop the execution at this point if this configuration is unsupported atm?